### PR TITLE
Added support for Multi-OS Engine 1.3.0-beta-2

### PR DESCRIPTION
**Pull request by liliomk**
[general]
- Updated moe-core and moe-ios jars to ones from MOE 1.3.0-beta-2
- Minor code cleanup
- Updated MOE versions to 1.3.0-beta-2
- Fixed bug in SharedLibraryLoader caused by change in MOE

[gdx-setup]
- Updated project files in GdxSetup.java
- Template build.gradle was updated and now properly handles resources via "ASSET_PATH"
- Deprecated files were removed
- Default and Icon images were moved to the appropriate location (Xcode target's folder)
- Updated Xcode project's layout

[test]
- Updated MOE test project
- Test now runs BulletTestCollection

### CODE REVIEW

**Comment by liliomk** [comment]
Please ignore [this pull request](https://github.com/libgdx/libgdx/pull/4519).

File: tests/gdx-tests-ios-intel/build.gradle [hunk 1]
```diff
@@ -15,34 +15,76 @@
  ******************************************************************************/
 
 buildscript {
+    repositories {
+        jcenter()
+        mavenLocal()
```
Comment (Tom-Ski) [inline tests/gdx-tests-ios-intel/build.gradle]: Why is maven local added here?

Comment (liliomk) [inline tests/gdx-tests-ios-intel/build.gradle]: We do this so we can use local builds of MOE to test.

Comment (Tom-Ski) [inline tests/gdx-tests-ios-intel/build.gradle]: Gotcha

**Pull request merged**